config/default/user.role.stanford_faculty.yml

```diff
@@ -1,11 +1,12 @@
 uuid: 2032bcd6-6126-490e-919b-e36722d00ddb
 langcode: en
 status: true
-dependencies: {  }
+dependencies: null
 _core:
   default_config_hash: 5Vm5wwYvL9cixzOqX2aUdfHLoZhPTJr0oHrLgP-vrV0
 id: stanford_faculty
 label: 'Stanford Faculty'
 weight: -5
 is_admin: null
-permissions: {  }
+permissions:
```

Comment:
No permissions should be for this role. it is meant to be just an identifier.

Reply:
Removed the permission, it was not being used because that functionality was reimplemented in another way.